src/openms/include/OpenMS/ANALYSIS/ID/SiriusAdapterAlgorithm.h

```diff
@@ -68,6 +68,16 @@ namespace OpenMS
       */
       static SiriusAdapterAlgorithm::SiriusTmpStruct constructSiriusTmpStruct();
 
+      /**
+      @brief Removes temporary directory and file structure
+
+      @param debug_level Debug level
+      @param tmp_dir_path Path to temporary directory
+      @param tmp_file_path Path to temporary file
+      */
+
+      static void removeSiriusTmp(const int& debug_level, const String& tmp_dir_path, const String& tmp_file_path);
```

Comment:
suggestion: pass ints by value

Reply:
I guess it is faster to use pass by value (also for primitive types such as bool and byte) since no dereferencing operation is needed - is that correct?

Reply:
yes, but the compiler might actually do other fancy things if it works on a copy (which is cheap in this case).